pandas/core/frame.py

```diff
@@ -6539,9 +6539,10 @@ def unstack(self, level=-1, fill_value=None):
     See Also
     --------
     %(other)s
-    pivot_table
-    DataFrame.pivot
-    Series.explode
+    pivot_table : Backward fill the new missing values in the resampled data.
```

Comment:
This description doesn't seem to be the one for `pivot_table`.

Reply:
@giovanism can you fix this up?